AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main.py

```diff
@@ -53,6 +55,7 @@ def wrap_run(cls, instance, request, workspace, editor, editor_test_results, lau
         for f in original_file_list:
             fm._restore_file(f, file_list[f])
 
+@pytest.mark.skip(reason="GHI #9722: Refactor/Reimplement Phyiscs AR Tests")
```

Comment:
Repeat Type: "Phyiscs"
```suggestion
@pytest.mark.skip(reason="GHI #9722: Refactor/Reimplement Physics AR Tests")
```